src/coreComponents/dataRepository/BufferOps_inline.hpp

```diff
@@ -1560,7 +1560,7 @@ Unpack( buffer_unit_type const * & buffer,
   {
     // Copy old capacities (no way to direct copy, must kernel launch)
     array1d< localIndex > newCapacities( var.size() );
-    forAll< parallelHostPolicy >( var.size(), [var, newCapacities = newCapacities.toView()]( localIndex const k )
```

Comment:
:facepalm: 
This was intended to be `var = var.toViewConst()`

Reply:
I like that better than the capture by reference I did.